src/sentry/management/commands/send_fake_data.py

```diff
@@ -23,7 +23,7 @@ def funcs():
     )
     loggers = itertools.cycle(["root", "foo", "foo.bar"])
     emails = itertools.cycle(["foo@example.com", "bar@example.com", "baz@example.com"])
-    timestamps = range(24 * 60 * 60)
+    timestamps = list(range(24 * 60 * 60))
     random.shuffle(timestamps)
```

Comment:
Using `random.sample` here might be more efficient:
```suggestion
    timestamps = random.sample(range(24 * 60 * 60, 24 * 60 * 60))
```

https://docs.python.org/3/library/random.html#random.sample
>To choose a sample from a range of integers, use a range() object as an argument. This is especially fast and space efficient for sampling from a large population: sample(range(10000000), k=60).

Reply:
I need to provide `k` to use this. Since this code can have an unbounded number of events, we'd end up just having k be the len anyway. I'm actually not sure why we don't just use `randint` here instead.

Reply:
>I need to provide k to use this. Since this code can have an unbounded number of events, we'd end up just having k be the len anyway.

That's already what I suggested, just forgot to add the `k=` part. The unbounded part is already handled by the `itertools.cycle()` call below.

>I'm actually not sure why we don't just use randint here instead.

Now, that makes a lot of sense (using `randint`).

Reply:
I guess the core difference is that with the list, we're guaranteed that we'll use all the timestamps before we rotate around again. `randint` still seems better, I'll just use that.